andela/technites-bn-backend
# #168049992 Integrate Coveralls code coverage service

#### What does this PR do?
  Integrate Coveralls code coverage service

#### Description of Task to be completed?
  Link Github app with code climate and coveralls web apps.
  Generate their budges to add to read me

#### How should this be manually tested?
  Check for budges on read me.
  Click on budges to see the assigned scores for both

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168049992
#### What are the relevant Github Issues?
#### Screenshots (if appropriate)
#### Questions:
